deploy/13-deploy-wstETH-weETH-oracles.ts

```diff
@@ -0,0 +1,45 @@
+import hre from "hardhat";
+import { DeployFunction } from "hardhat-deploy/dist/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { ADDRESSES } from "../helpers/deploymentConfig";
+
+const func: DeployFunction = async function ({ getNamedAccounts, deployments, network }: HardhatRuntimeEnvironment) {
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+  const proxyOwnerAddress = ADDRESSES[network.name].timelock;
+  const { WETH, wstETH, weETH } = ADDRESSES[network.name];
+
+  const resilientOracle = await hre.ethers.getContract("ResilientOracle");
+  const chainlinkOracle = await hre.ethers.getContract("ChainlinkOracle");
+
+  await deploy("wstETHOneJumpChainlinkOracle", {
+    contract: "OneJumpOracle",
+    from: deployer,
+    log: true,
+    deterministicDeployment: false,
+    args: [wstETH, WETH, resilientOracle.address, chainlinkOracle.address],
+    proxy: {
```

Comment:
We are setting the property `viaAdminContract` in the rest of the oracle deployments. Shouldn't we set it here too? /cc @GitGuru7

Reply:
The viaAdminContract is required for deploying upgradeable contracts on the zkSync network, and this specific oracle is deployed exclusively on Arbitrum, see [here](https://github.com/VenusProtocol/oracle/pull/214/files/f698c0c0843d58db57b8f50baa3b1d049edf670c#diff-860976e7cae6103ad5f743d19283d4fbb602120fa1cab425c0358a86a9cc09f0R43),  Therefore, zksync related changes are unnecessary .

Reply:
Again it is a choice. For the sake of consistency, we might need to update everywhere. Or just update this when needed in a zksync context. @chechu wdyt?

Reply:
I would use `OptimizedTransparentUpgradeableProxy` and the `viaAdminContract` option, to be consistent with the deployments to zkSync (maybe in the future we need to deploy these contracts to that network). I wouldn't change the previous deployment scripts using `OptimizedTransparentProxy` now, to limit the scope of the change

Reply:
Deployed oracles with OptimizedTransparentUpgradeableProxy using viaAdminContract, https://github.com/VenusProtocol/oracle/commit/a9966bc35c905f1bc84009af4f8a728ba2e1d98e